orunmila.go

```diff
@@ -245,6 +245,8 @@ func searchWordsByTagIds(db *sql.DB, tags string) {
 	}
 	err = rows.Err()
 	check(err)
+
+	db.Close()
```

Comment:
how about we close the db where it was opened instead? this will cause us trouble in the future

Reply:
com'on at least appreciate the aesthetically-placed copy pasted `db.Close()` around

Reply:
i do :smile: